rewrite-test/src/main/kotlin/org/openrewrite/java/AddImportTest.kt

```diff
@@ -21,33 +21,69 @@ import org.openrewrite.whenParsedBy
 
 interface AddImportTest: RefactorVisitorTest {
     @Test
-    fun addMultipleImports(jp: JavaParser) {
-        "class A {}"
-                .whenParsedBy(jp)
-                .whenVisitedBy(AddImport().apply { setType("java.util.List"); setOnlyIfReferenced(false) })
-                .whenVisitedBy(AddImport().apply { setType("java.util.Set"); setOnlyIfReferenced(false) })
-                .isRefactoredTo("""
-                    import java.util.List;
-                    import java.util.Set;
-                    
-                    class A {}
-                """)
-    }
+    fun addMultipleImports(jp: JavaParser) = assertRefactored(
+            jp,
+            visitors = listOf(
+                    AddImport().apply { setType("java.util.List"); setOnlyIfReferenced(false) },
+                    AddImport().apply { setType("java.util.Set"); setOnlyIfReferenced(false) }
+            ),
+            before = """
+                class A {}
+            """,
+            after = """
+                import java.util.List;
+                import java.util.Set;
+    
+                class A {}
+            """
+    )
 
     @Test
-    fun addNamedImport(jp: JavaParser) {
-        "class A {}"
-                .whenParsedBy(jp)
-                .whenVisitedBy(AddImport().apply { setType("java.util.List"); setOnlyIfReferenced(false) })
-                .isRefactoredTo("""
-                    import java.util.List;
-                    
-                    class A {}
-                """)
-    }
+    fun addNamedImport(jp: JavaParser) = assertRefactored(
+            jp,
+            visitors = listOf(
+                AddImport().apply { setType("java.util.List"); setOnlyIfReferenced(false) }
+            ),
+            before = "class A {}",
+            after = """
+                import java.util.List;
+                
+                class A {}
+            """
+    )
+
+    @Test
+    fun lastImportWhenFirstClassDeclarationHasJavadoc(jp: JavaParser) = assertRefactored(
+            jp,
+            visitors = listOf(
+                AddImport().apply {
+                    setType("java.util.Collections")
+                    setStaticMethod("*")
+                    setOnlyIfReferenced(false)
+                }
+            ),
+            before = """
+                import java.util.List;
+                
+                /**
+                 * My type
+                 */
+                class A {}
+            """,
+            after = """
+                import java.util.List;
+                
+                import static java.util.Collections.*;
+                
+                /**
+                 * My type
+                 */
+                class A {}
+            """
+    )
 
     @Test
-    fun lastImportWhenFirstClassDeclarationHasJavadoc(jp: JavaParser) {
+    fun lastImportWhenFirstClassDeclarationHasJavadoc2(jp: JavaParser) {
```

Comment:
Looks like this was left over on accident. There should only be one `lastImportWhenFirstClassDeclarationHasJavadoc`, right?